src/posting/app.py

```diff
@@ -701,6 +702,52 @@ def on_mount(self) -> None:
         )
         self.theme_change_signal = Signal[ColorSystem](self, "theme-changed")
         self.theme = self.settings.theme
+        self.try_to_add_xresources_theme()
```

Comment:
I'd want to put this behind a config option `use_xresources` which is `False` by default. If that option is `True`, then this method would be called.

It should also be possible to set the theme to `xresources-dark` or `xresources-light` directly from the config file (e.g. `theme: xresource-dark`).